.storybook/components/random-highlight/RandomHighlight.tsx

```diff
@@ -22,36 +21,28 @@ const highlightColors = [
 
 type PickRandomItem<T> = (array: T[]) => T;
 
-const pickRandomItem: PickRandomItem<string> = array =>
+export const pickRandomItem: PickRandomItem<string> = array =>
```

Comment:
Consider moving this to a shared util (e.g. `./storybook/utils/highlights.ts`) and then import from stories that need it. This avoids unnecessary cross-dependencies between stories.